package/run.ps1

```diff
@@ -46,6 +46,40 @@ function Get-StringHash {
     return (Get-FileHash -InputStream $stringAsStream -Algorithm SHA256).Hash.ToLower()
 }
 
+# stops the RKE2 process as well as other processes spawned from rke2
+function Stop-RKE2-Processes () {
+    $rke2ProcessNames = @('rke2', 'kube-proxy', 'kubelet', 'containerd', 'calico-node')
+    foreach ($processName in $rke2ProcessNames) {
+        Write-LogInfo "Checking if $processName process exists"
+        if (Get-Process -Name $processName -ErrorAction SilentlyContinue) {
+            Write-LogInfo "$processName process found, stopping now"
+            Stop-Process -Name $processName
+        }
+    }
+
+    # ensure all processes have exited
+    $waitLimit = 3
+    foreach ($processName in $rke2ProcessNames) {
+        # check if the process still exists
+        Get-Process -Name $processName -ErrorAction SilentlyContinue -ErrorVariable processNotFoundError
```

Comment:
Should this be:

```suggestion
        Get-Process -Name $processName -ErrorAction SilentlyContinue -ErrorVariable $processNotFoundError
```

Reply:
`ErrorVariable` does not need the `$` https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-7.3#-errorvariable